packages/typespec-ts/test/modularIntegration/generated/lro/modular/src/StandardClient.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ClientOptions } from "./common/interfaces.js";
+import {
+  User,
+  createOrReplace,
+  beginCreateOrReplaceAndWait,
+  beginCreateOrReplace,
+  CreateOrReplaceOptions,
+  createStandard,
+  StandardContext
+} from "./api/index.js";
+import { OperationState, SimplePollerLike } from "@azure/core-lro";
+
+export class StandardClient {
+  private _client: StandardContext;
+
+  /** Illustrates bodies templated with Azure Core with long-running operation */
+  constructor(options: ClientOptions = {}) {
+    this._client = createStandard(options);
+  }
+
+  createOrReplace(
+    role: string,
+    name: string,
+    options: CreateOrReplaceOptions = { requestOptions: {} }
+  ): Promise<User> {
+    return createOrReplace(this._client, role, name, options);
+  }
+
+  /**
+   * Below code are manually written now and will be generated later
+   */
+  beginCreateOrReplaceAndWait(
```

Comment:
We decided to generate this operation in Track 2 management-plane libraries to accommodate customers who are accustomed to the Track 1 pattern of waiting operations. If this generator is meant for a different segment of customers, I would strongly be in favor of removing this operation.

Reply:
In latest design we remove this `beginCreateOrReplaceAndWait` operation,https://github.com/Azure/autorest.typescript/pull/1905.